analysis/plotting.py

```diff
@@ -16,13 +16,16 @@
 import numpy as np
 import Orange
 import seaborn as sns
+import warnings
 
 from matplotlib import colors
 from matplotlib import pyplot as plt
 from analysis import data_utils
 
 _DEFAULT_TICKS_COUNT = 12
 _DEFAULT_LABEL_ROTATION = 30
+warnings.simplefilter(action='ignore', category=FutureWarning)
```

Comment:
I think the correct move is to fix what these things are complaining about instead of ignoring them.